software/zoraxy.yml

```
@@ -0,0 +1,11 @@
+name: "Zoraxy"
+website_url: "https://zoraxy.aroz.org/"
+source_code_url: "https://github.com/tobychui/zoraxy"
+description: "General purpose HTTP reverse proxy and forwarding tool."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Go
+  - Docker
+tags:
+  - Web Servers
```

```suggestion
tags:
  - Web Servers
  - Analytics
```

Built-in analytics tool

---

The HAProxy stats page shows system metrics like session rates, bytes transferred, denied and errored requests... it's more like a monitoring endpoint. Zoraxy's built-in statistics seem to provide web log analytics, similar to GoAccess or other web log analytics tools, like user agent, country, traffic graph... Anyway it's a minor feature, I'll aprove the PR as-is - whether or not you add the `Analytics` tag is up to you.